backend/app/services/implementations/log_records_service.py

```diff
@@ -42,8 +42,7 @@ def to_json_list(self, logs):
                     {
                         "log_id": log[0],
                         "employee_id": log[1],
-                        "resident_first_name": log[2],
-                        "resident_last_name": log[3],
+                        "resident_id": log[2],
                         "datetime": str(log[4].astimezone(timezone("US/Eastern"))),
```

Comment:
it goes from resident_id = log[2] to datetime = str(log[4].astimezone(timezone("US/Eastern"))). log[3] is missing :(((( pretty sure thats gonna mess something up